### DISCUSSION TITLE
Postgis sync on QFieldCloud on-premise instance

### DISCUSSION THREAD

**Discussion by danieluct** [opengisch/QField#3494] [Q&A]
Hello,
We deployed the latest release of QFieldCloud on a test server. While editing and syncing works as expected when using geopackages, we were unable to perform a sync with the database. The conversion process seems to remove all references to the database. Note that we tried both the saved user/pass approach and the secrets approach (we exposed the Secret object in the Django app in order to do so). We also manually tested the connection between the docker instances and the database server and everything seems to be OK.

Any hint how to enable syncing with the database? Is postgis syncing open-sourced or it is a proprietary feature of app.qfield.cloud ?

Thank you.

**Comment by suricactus** [comment]
How do you upload you project to you local QFieldCloud instance? There are two options when the upload project screen. Please make sure you are using "Create a new empty QFieldCloud project" for such cases as described here https://docs.qfield.org/get-started/tutorials/get-started-qfc/#create-and-configure-your-cloud-project .
![image](https://user-images.githubusercontent.com/2820439/197488497-00055866-0880-4526-8b9d-522e7b6b3cab.png)

**Reply by danieluct** [reply]
Thank you. If I create a new empty project and then manually copy the project files and afterwards upload it through the QGIS plugin, the sync with the db works (for the moment I tested only with user/pass saved within project).

**Reply by geogeek11** [reply]
Hello 
I have tried the same way but it did not work, is there any way to make the sync to postgis DB using on premise setup ?

**Reply by mahmoodi-source** [reply]
Hello guys, could you please guide me through the processes? I'm unsure about how you connected your database to the locally saved files.

**Reply by mahmoodi-source** [reply]
Hey guys, I have a problem uploading project files to an existing empty project created on our QfieldCloud server. When I paste the project files in the local directory, everything is good, and the message says that "the entered path contains one QGIS project file. Exactly as it should be."

Now, when I sync the project through Qfield Sync, at the end, it will give me the error of failing to upload multiple projects.

and when I create a project through Qfield sync plugins in our QfieldCloud server and then upload the uploaded files in the directory, it uploads without problem, but it cannot be downloaded in the Qfield app and gives me the following error:
![image](https://github.com/opengisch/QField/assets/144428014/df3c78f8-356e-4d3a-b42d-9a1af3f234e6)

I don't know what the problem is.

**Reply by suricactus** [reply]
Hello, please do not write the same message over all QFieldCloud related forums without interlinking them. Note original post was here: https://github.com/opengisch/qfieldcloud/issues/870 . I have already answered there, it is misconfiguration of your plans.

If you are interested for dedicated assitance in configuring QFieldCloud on-premise, please write us on sales at qfield.cloud .
